crates/services/gas_price_service/src/v1/da_source_service.rs

```diff
@@ -2,11 +2,13 @@ use crate::v1::da_source_service::service::DaBlockCostsSource;
 use std::time::Duration;
 
 pub mod block_committer_costs;
+#[cfg(test)]
 pub mod dummy_costs;
 pub mod service;
 
 #[derive(Debug, Default, Clone, Eq, Hash, PartialEq)]
 pub struct DaBlockCosts {
+    pub sequence_number: u32,
```

Comment:
It would be better if it was `bundle_id` or if we somehow connected it to the block committer(comment can help)=)